src/Sistrategia.SAT.CFDiWebSite/Models/ComprobanteViewModels.cs

```diff
@@ -666,6 +666,7 @@ public class ComprobanteHtmlViewModel
         public string Notas { get; set; }
 
         public string MetodoDePago { get; set; }
+        public string MetodoDePagoDisplayName { get; set; }
```

Comment:
Justamente aquí es donde podríamos aprovechar para hacer el concatenado, podríamos agregar el campo Description y hacer la concatenación como una fórmula local:
public string MetodoDePagoCode {get;set;}
public string MetodoDePagoDescription {get;set;}
public string MetodoDePagoDisplayName { 
  get { return this.MetodoDePagoCode + "-" + this.MetodoDePagoDescription; }
}
El código es de ejemplo, simplificado, no considera que vengan vacíos los valores etc. Pero básicamente se podrían agregar las verificaciones que se hacen en ComprobanteController y así se reutiliza más código y todo queda más limpio.